js/pulse.js

```diff
@@ -362,11 +363,13 @@
                     log('No Pulse hostname found in plugin parameters or media metadata; will not attempt to show Pulse ads');
                     noPulseConfiguration = true;
                     this.ready = true;
+                    amc.reportPluginLoaded(Date.now() - this.initTime, this.name)
```

Comment:
The other plugins report plugin loaded after calling amc.onAdManagerReady. Should this plugin do the same for consistency?

Reply:
Fixed